stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: postgresql
-version: 0.9.3
+version: 0.9.4
+appVersion: 0.9.4
```

Comment:
@AdamDang the appVersion should reflect the version of the psql server installed by the chart which is `9.6.2`